src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Calculator.css';
+
+const numSigns = [
+  'AC',
+  '+/-',
+  '%',
+  '/',
+  7,
+  8,
+  9,
+  '*',
+  4,
+  5,
+  6,
+  '-',
+  1,
+  2,
+  3,
+  '+',
+  0,
+  '.',
+  '=',
+];
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
+      <div className="cal-container">
+        <div className="display">0</div>
```

Comment:
- [ ] [Optional] [Highly recommended] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=the%20next%20project.-,Use%20functional%20components,-Create%20at%20least), Please make the Calculator component a functional component again.

### ⚠️I marked this as optional because the last reviewer didn't pay attention to the new requirement for this particular project so please consider doing the change⚠️